jobrunner/cli/manifests.py

```diff
@@ -0,0 +1,94 @@
+"""
+Ops utility for backfilling manifest.json files from db
+"""
+import argparse
+
+from jobrunner.executors import local
+from jobrunner.lib import database
+from jobrunner.models import Job
+
+
+def main(workspaces=None):
+    conn = database.get_connection()
+    workspaces = [
+        w["workspace"] for w in conn.execute("SELECT DISTINCT(workspace) FROM job;")
+    ]
+
+    n_workspaces = len(workspaces)
+    for i, workspace in enumerate(workspaces):
+        print(f"workspace {i+1}/{n_workspaces}: {workspace}")
+
+        level4_dir = local.get_medium_privacy_workspace(workspace)
+
+        sentinel = level4_dir / ".manifest-backfill"
+        if sentinel.exists():
+            print(" - already done, skipping")
+            continue
+
+        write_manifest(workspace)
```

Comment:
There's a potential race-condition here if the main job-runner loop is itself in the process of updating the manifest. Is it feasible to stop job-runner while the backfill runs? Do you have a guess for how long it will take?

Reply:
That was my plan, given how busy we are. I don't expect it take more than a couple of hours at most.